spring/src/main/java/io/seata/spring/annotation/GlobalTransactionScanner.java

```diff
@@ -297,4 +303,20 @@ public void setApplicationContext(ApplicationContext applicationContext) throws
         this.setBeanFactory(applicationContext);
     }
 
+    @Override
+    public Object postProcessAfterInitialization(Object bean, String beanName) throws BeansException {
+        if (bean instanceof DataSource && ConfigurationFactory.getInstance().getBoolean(DATASOURCE_AUTOPROXY, false)) {
+            Map beansOfTypeMaps = applicationContext.getBeansOfType(DataSourceProxy.class);
+            if (beansOfTypeMaps.size() < 1 && applicationContext instanceof ConfigurableApplicationContext) {
```

Comment:
just support one DataSourceProxy？

Reply:
> just support one DataSourceProxy？

For now,yes,but i will re-design it to support multi-datasource.